### ISSUE TITLE
Blink or dialog for new alerts ?

### ISSUE THREAD

**Issue by duylong** [statusengine/interface#8]
Hi,

Sometimes we do not see the alerts arrived in the dashboard, do you think that a system of blinking (dialog) for a predefined time is conceivable?

**Comment by nook24** [comment]
Do you mean the Dashboard?
![bildschirmfoto 2017-09-18 um 20 31 19](https://user-images.githubusercontent.com/9019992/30557950-2af674de-9cb0-11e7-8c80-bb92dbace685.png)

Or the Issues view?
![bildschirmfoto 2017-09-18 um 20 31 24](https://user-images.githubusercontent.com/9019992/30557976-3c27e774-9cb0-11e7-954f-839ae8f74095.png)

**Comment by duylong** [comment]
All ^^

By experience, we are more reactive when a few things move / blink on the screen. The interest is to know what has just happened if we quickly look elsewhere.

Nagvis uses a flashing system and I like :)

**Comment by nook24** [comment]
I added a notification to the dashboard, when every a host or service change it's state. (No notifications for recovery)
![bildschirmfoto 2017-09-19 um 20 34 20](https://user-images.githubusercontent.com/9019992/30609112-c321a4c0-9d7a-11e7-8c13-7cc5f05c6b94.png)

I tested this on the issues view, but this is super annoying, so I will only add this to the dashboard :)

PS: I personally like Slack (or something similar) as notification method

**Comment by duylong** [comment]
Some quick suggestions:

* Customize the refresh time and layout position
* lowercase "objectType"

PS: I am not sure that all people like to make the messages pass through the slack servers, I use aNag for my part O:)

**Comment by nook24** [comment]
I switched to lower case and you can now change the auto interval to predefined values.

Changeable layout options is bit to much overhead.

**Comment by duylong** [comment]
Can we also custom the notification refresh time ?

**Comment by nook24** [comment]
The dashboard just compare the old values and the new values. If there are more services are critical, than in the last check, the interface will throw the notification.
So the refresh time is directly set with the refresh time of the dashboard.

**Comment by duylong** [comment]
```
        var notify = function (number, objectType, state, type) {
            noty({
                theme: 'metrouiAdminLTE',
                progressBar: true,
                layout: 'topRight',
                type: type, // success, error, warning, information, notification
                text: String(number) + ' ' + objectType + ' changed to state ' + state,
??? =>          timeout: 2500,
                animation: {
                    open: 'animated flipInX',
                    close: 'animated flipOutX'
                }
            });
        };
```
So what is this time?

**Comment by nook24** [comment]
This is the delay for closing the notification (in ms). So it will be visible for 2.5 seconds.

**Comment by duylong** [comment]
Aaaah, that's what I wanted to say, I would like to increase the time without modifying the code ^^

**Comment by nook24** [comment]
So, what's the time you would prefer?^^

**Comment by duylong** [comment]
The time delay for closing the notification^^

**Comment by nook24** [comment]
Yes i know^^
I don't want to implement to much settings to the interface, so the question is, what is the time you would prefer as default closing timeout

**Comment by duylong** [comment]
I would like to test with 10s, what do you think?

**Comment by nook24** [comment]
I set the timeout to 7,5 secs. Merged into Ui-3.2.0
Full changelog: https://statusengine.org/roadmap/#ui-3.2.0
